### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by forvaidya**
Improve README.md for windows.
I have edited English version only as I don't know other languages and localizations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,12 +48,14 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 ## Running the website locally using Hugo
 
+Windows 10, make is not available by default. To run hugo without container support,
 Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```
Comment (sftim) [inline README.md:52]:
```suggestion
You can run Hugo directly (not inside a container). You need to install the version ofHugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```

File: README.md [hunk 2]
```diff
@@ -48,12 +48,14 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
...
 
 To build and test the site locally, run:
 
 ```bash
-make serve
+hugo server --buildFuture
+#Listens on port 1313,
```
Comment (sftim) [inline README.md:58]:
```suggestion
make serve

# If you are using a platform such as Windows that does not provide the
# "make" utility, manually verify that you have the dependencies in place
# and then run:
hugo server --buildFuture
```

File: README.md [hunk 3]
```diff
@@ -48,12 +48,17 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 ## Running the website locally using Hugo
 
-Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run Hugo directly (not inside a container). You need to install the version ofHugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```
Comment (kbhawkey) [inline README.md:51]: I think the original text is clear (install the extended version).
Is it confusing that the sentence points the reader to the `netlify.toml` file?

File: README.md [hunk 4]
```diff
@@ -48,12 +48,17 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
...
 
 To build and test the site locally, run:
 
 ```bash
 make serve
+
```
Comment (kbhawkey) [inline README.md:57]: What do you think about listing the different commands in separate code blocks or list only
`hugo server --buildFuture`? The `make serve` target performs an additional check.

**Comment by kbhawkey** [comment]
Related #22995.

File: README.md (hunk 4 above)
Comment (sftim) [reply README.md:57]: That extra check from `make serve` is useful and if available I'd steer readers to use it.

File: README.md (hunk 3 above)
Comment (sftim) [inline README.md:51]: I saw a typo (sorry)
```suggestion
You can run Hugo directly (not inside a container). You need to install the version of Hugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```

File: README.md [hunk 5]
```diff
@@ -48,12 +48,17 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 ## Running the website locally using Hugo
 
-Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run Hugo directly (not inside a container). You need to install the version of Hugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```
Comment (kbhawkey) [inline README.md:51]: HI @forvaidya .
I think this change needs to mention that the extended version of Hugo is required.
Hugo publishes more than one type of a given version.
From the theme's documentation:
```
Install a recent release of the Hugo "extended" version (we recommend version 0.53 or later).
```

**Comment by irvifa** [comment]
Hi @forvaidya do you still working on this? 👋

**Comment by kbhawkey** [comment]
Hello @forvaidya .
I plan on closing this pull request.
Feel free to reopen this PR to make further updates.
/close

**Pull request closed without merging**